components/data_comps/datm/bld/namelist_files/namelist_definition_datm.xml

```diff
@@ -4,32 +4,93 @@
 
 <namelist_definition>
 
-<!-- =========================================  -->
-<!--- DATM env settings                         -->
-<!-- =========================================  -->
+<!-- ========================================================================================  -->
+<!-- Stream file variables                                                                     -->
+<!-- ========================================================================================  -->
 
-<entry id="DATM_MODE"
-type="char*256"
-category="datm_setting"
-group="datm_env_settings" 
-valid_values="CLM_QIAN,CLM1PT,CLMCRUNCEP,CLMCRUNCEP_V5,CLMGSWP3,CPLHIST3HrWx,CORE2_NYF,CORE2_IAF,COPYALL_NPS_v1,COPYALL_CORE2_v1,WW3,NLDAS">
-Data Model mode to process forcing data
+<entry id="streamslist"
+type="char*1024(100)"
+category="streams"
+group="streams_file">
+List of streams used for the given datm_mode.
 </entry>
 
-<entry id="DATM_PRESAERO"
-type="char*256"
-category="datm_setting"
-group="datm_env_settings"
-valid_values="none,pt1_pt1,clim_2000,clim_1850,trans_1850-2000,rcp2.6,rcp4.5,rcp6.0,rcp8.5">
-Prescribed aerosol mode
+<entry id="strm_domdir"
```

Comment:
@jedwards4b The `namelist_definition` code wouldn't really need to change all that much for what you're suggesting. I think that pretty much all of the differences would be in lines 92-122 of the file. However, changing the XML files themselves would take a bit longer; you'd presumably want a conversion utility for that.